scripts/s3mi

```diff
@@ -338,7 +338,8 @@ def adjust_RAM_params():
         MAX_CONCURRENT_REQUESTS = 15
 
 
-def get_credentials():
+def get_credentials(quiet=False,
+                    DISABLED={"vars": {}, "expiration_time": time.time() + 365*24*3600}):  # This result effectively disables credentials caching/update.
```

Comment:
I don't know if this is intentional, but assigning DISABLED here makes expiration_time to be a fixed point in time, since this code gets executed during library initialization.
That could be a problem if this package were being imported by any other long running tools (by long running I mean 1 year or more), but I guess this is not a problem in this context, correct?

Reply:
I also noticed this default field is not being passed anywhere else. Do you feel it would make sense to make this a variable/constant instead of a default parameter? This is probably more an stylistic question, since both approaches would work similarly from a functional perspective.

Reply:
Good catch about the expiration time, it was chosen far enough in the future that it shouldnt' matter.  If a single s3mi invocation lasts more than 1 year, it's got other problems.

This is the python way of making what C++ would call a static variable with function scope.   The python language doesn't actually offer a different way of doing that, so it's pretty amusing all linters call it poor practice.